### PR TITLE
clarify when work should be in separate organisations

### PR DESCRIPTION
**Pull request by sbfnk**
The manual currently states that work done by the group should be in the `epiforecasts` organisation. This is already not the case for the [modelling/scenario hubs](https://github.com/covid19-forecast-hub-europe/) and might also not be a good choice for the `epinowcasts` package (see [related discussion](https://github.com/epiforecasts/epinowcast/discussions/161)). This PR aims to clarify when projects should be in separate organisations.

### CODE REVIEW

File: lab-manual.qmd
Comment (Bisaloo) [inline lab-manual.qmd]:
```suggestion
The group’s github repository is <https://github.com/epiforecasts>. Initial and exploratory work should be in personal repositories; maturing and collaborative group projects should be in the group repository. Projects that involve multiple repositories and/or include core developers from outside the group can live within separate organisations and mention the affiliation to the group in some other way.
```

I definitely agree with the general feeling. I would probably say "can live" instead of "should live", to ultimately leave it up to the maintainer team but no strong feelings on this.

**Review by seabbs — APPROVED** [review]
LGTM

**Pull request merged**
